README.md

```diff
@@ -33,11 +33,12 @@ For development purposes, this repo is recommended as it includes unit tests, to
 #### macOS
 
 1. Mac only: [install XCode](https://apps.apple.com/us/app/xcode/id497799835)
-2. XCode command line tools (xcode-select --install) are installed
+2. Install dependencies: `brew bundle`
+3. XCode command line tools (`xcode-select --install`) are installed
```

Comment:
- Can we keep XCode and XCode command line tools together? We can move `brew bundle` up to 1. or down to 3.
- Unrelated to this diff but can you remove the `Mac only:` prefix, now that we have a dedicated macOS block?